src/api/auth.js

```diff
@@ -128,6 +128,34 @@ export function upgradeFacilitatorAccessToken(facilitatorAccessToken : string, {
     });
 }
 
+export function upgradeFacilitatorAccessTokenWithIgnoreCache(facilitatorAccessToken : string, { buyerAccessToken, orderID } : {| buyerAccessToken : string, orderID : string |}) : ZalgoPromise<void> {
```

Comment:
Lets keep a flat params for this method siganture as all params for mandatory for this. 
```suggestion
export function upgradeFacilitatorAccessTokenWithIgnoreCache(facilitatorAccessToken : string, buyerAccessToken : string, orderID : string) : ZalgoPromise<void> {
```